src/ONYXKEYS.js

```diff
@@ -171,6 +171,7 @@ export default {
     FORMS: {
         ADD_DEBIT_CARD_FORM: 'addDebitCardForm',
         REQUEST_CALL_FORM: 'requestCallForm',
+        COMPANY_STEP_FORM: 'companyStepForm',
```

Comment:
We should use `REIMBURSEMENT_ACCOUNT_FORM` for all forms in the VBA flow.

Reply:
fixed